### ISSUE TITLE
Debian: xline.db directory is not created upon package installation.

### ISSUE THREAD

**Issue by dijit** [darkscience/ds-salt#10]
`Wed Sep 16 09:09:45 2015: ANNOUNCEMENT: database: cannot create new db: No such file or directory (2)`

Related: https://github.com/inspircd/inspircd/issues/982

`/quote unloadmodule m_xline_db.so
~:# mkdir -p /usr/lib/inspircd/data
/quote loadmodule m_xline_db.so`

is the manual fix.

**Comment by Dracozny** [comment]
This doesn't seem to work for me. running ubuntu 14.04 and InspIRCd-2.0.5

**Comment by dijit** [comment]
@Dracozny What is the error you're getting? permission denied?

**Comment by Dracozny** [comment]
Correct. I already followed the steps you suggested. Verified folder
permissions. Still get the same permission denied error.

**Comment by dijit** [comment]
Not `no such file or directory`, actual permission denied?

try: `~:# chown ircd: -R /usr/lib/inspircd/data`

or please paste the error. :)

**Comment by Dracozny** [comment]
the user is "irc", there is no "ircd" in /etc/passwd

As said permissions are all correct are you sure this shouldn't be in
/var/lib?
<quote>
root@bbb:~# ls -l /usr/lib/inspircd/
total 28
drwxr-xr-x 2 irc irc  4096 May 11 16:52 data
-rwxr-xr-x 1 irc irc 11892 Mar 29 10:55 inspircd
drwxr-xr-x 2 irc irc 12288 May  9 19:02 modules
</quote>

**Comment by Dracozny** [comment]
I think I found my problem. difference of a setting between versions. older
version doesn't specify xline file directory.
